src/views/GovernanceTaskList/index.tsx

```diff
@@ -42,11 +43,81 @@ const GovernanceTaskList = () => {
   );
 
   const intakeStatus = intakeStatusFromIntake(systemIntake);
-  const intakeLink = chooseIntakeLink(systemIntake, intakeStatus);
+  const intakeLink = () => {
+    const link = chooseIntakeLink(systemIntake, intakeStatus);
+    switch (intakeStatus) {
+      case 'COMPLETED':
+        return <Link to={link}>View Submitted Request Form</Link>;
+      case 'CONTINUE':
+        return (
+          <UswdsLink className="usa-button" variant="unstyled" href={link}>
+            Continue
+          </UswdsLink>
+        );
+      case 'START':
+        return (
+          <UswdsLink className="usa-button" variant="unstyled" href={link}>
+            Start
+          </UswdsLink>
+        );
+      default:
+        return null;
+    }
+  };
+
   const intakeFeedbackStatus = feedbackStatusFromIntakeStatus(
     systemIntake.status
   );
-  const businessCaseStatus = bizCaseStatus(intakeStatus, businessCase);
+  const intakeFeedbackStatusBanner = () => {
+    if (
+      systemIntake.status === 'CLOSED' ||
+      systemIntake.status === 'APPROVED'
+    ) {
+      return (
+        <Alert type="info" slim>
+          Please check your email for feedback and next steps.
+        </Alert>
+      );
+    }
+    return null;
+  };
+
+  const businessCaseStatus = bizCaseStatus(systemIntake.status, businessCase);
+  const businessCaseLink = () => {
+    const link = chooseBusinessCaseLink(businessCase, intakeStatus);
+    if (!link) {
```

Comment:
In what situations is the URL for the link likely to not exist yet? Might be worth a comment?